src/lib.rs

```diff
@@ -506,6 +512,23 @@ note: skipped the following crates since they have no library target: {skipped}"
                             })?;
                             Ok((crate_name.clone(), None))
                         } else {
+                            let package_overrides =
+                                manifest::deserialize_lint_table(selected.metadata.clone())
+                                    .with_context(|| {
+                                        format!(
+                                    "{} [package.metadata.cargo-semver-checks] table is incorrect",
```

Comment:
Idea: do we by any chance know the path to the `Cargo.toml` file here? If it isn't too difficult, it might be useful to not just say which crate had the invalid table but also give a path to it.

This is because in some cases, projects may contain more than one crate by the same name — for example, see #462. In such cases, knowing the `Cargo.toml` path would be very helpful to the user.

If it's too hard to do now, we can just open a `C-enhancement` issue for the idea and tackle it another time.